build.sbt

```diff
@@ -44,21 +44,6 @@ lazy val allSettings = Seq(
 
 lazy val example = project
 
-lazy val protobufSettings = Seq(
```

Comment:
I have these settings in several projects that I copy-paste around, also in some cases I have protobuf settings across several projects. In metadoc it's fine to inline it.